src/app/components/JumpTo/index.tsx

```diff
@@ -46,7 +46,22 @@ const JumpTo = ({ jumpToHeadings }: JumpToProps) => {
     clickTrackerHandler(e);
     setHash(subheadingId);
   };
-
+  const headingsToRender = [
+    // Add article subheadings into a copy of the array to stop mutation of the original array
+    ...jumpToHeadings.map(({ heading }) => ({
+      heading,
+      id: idSanitiser(heading),
+    })),
+    // Related content link also added to the headings list when that OJ is present on the page
+    ...(showRelatedContentLink
+      ? [
+          {
+            heading: relatedContent,
+            id: 'section-label-heading-related-content-heading',
+          },
+        ]
+      : []),
```

Comment:
Is this a wee bit more readable?

```suggestion
    ...(showRelatedContentLink && 
          {
            heading: relatedContent,
            id: 'section-label-heading-related-content-heading',
          },
      ),
```

Reply:
When I tried this way I get the error 'Type 'false | { heading: string; id: string; } | undefined' must have a 'Symbol.iterator' method that returns an iterator.'.  With a further explanation being 'The problem is that the spread operator is being used on a conditional object, which can result in a non-iterable value. To fix this, you should conditionally concatenate the related content link to the array.'

Might be easier to just leave it so I can get this merged for an a11y swarm later

Reply:
Oops sorry didn't realise that wouldn't work - this is fine, happy to leave as is 👍